### PR TITLE
PactCommand Class Usage Example

### PR DESCRIPTION
**Pull request by Luzzotica**
## Reason:

I agree with Pascal on having a lighter weight solution to writing and executing pact code.
So, I've added docs to PactCommand and a usage example to the readme.
This closes out #161 

## Changes made (preferably with images/screenshots):

PactCommand usage documentation.
Screen Shot 2023-02-06 at 4 01 37 PM

I've also added documentation to each of the functions in the PactCommand class.

## Check off the following:

- [x] I have reviewed my changes and run the appropriate tests.
- [x] I have have run `rush change` to add the appropriate change logs.
- [x] I have added/edited docs.
- [x] I have added tutorials.
- [x] I have double checked and DEFINITELY added docs.

### CODE REVIEW

File: packages/libs/client/README.md [hunk 1]
```diff
@@ -376,6 +377,41 @@ const commandBuilder = kadenaCoinTemplates['safe-transfer']({
...
+# Using the PactCommand class
+
+If you don't wish to generate JS code for your contracts, or use templates, you can use the PactCommand class directly to build a command modularly, and then easily send it.
+
+```ts
+import { PactCommand } from '@kadena/client';
+
+var pactCommandBuilder = new PactCommand()
+  .addData({ // Add environment data to the transaction
+    person: "Randy",
```
Comment (Randynamic) [inline packages/libs/client/README.md:389]: 😆 +1 for adding Randy 😉

**Review by Randynamic — APPROVED** [review]

**Review by ash-vd — APPROVED** [review]

**Comment by alber70g** [comment]
Maybe add sign command in the example? With signWithChainweaver?

**Review by alber70g — APPROVED** [review]

**Review by nillo — APPROVED** [review]

**Comment by Randynamic** [comment]
> Maybe add sign command in the example? With signWithChainweaver?

IMHO I think we should keep it clean as we are also going to add different signing methods. This PR is about PactCommand, not the signing part.

**Review by Randynamic — APPROVED** [review]

**Comment by Randynamic** [comment]
@Luzzotica can you run a `rush change`?

**Comment by alber70g** [comment]
> > Maybe add sign command in the example? With signWithChainweaver?
> 
> IMHO I think we should keep it clean as we are also going to add different signing methods. This PR is about PactCommand, not the signing part.

Agreed!

**Pull request merged**